functions/__fisher_path_update.fish

```diff
@@ -5,7 +5,7 @@ function __fisher_path_update -a path
         git rebase --abort
         git fetch --quiet origin master
         git reset --quiet --hard FETCH_HEAD
-        git clean --quiet -d --force
+        git clean --quiet -dx --force
```

Comment:
yup!

> On Mar 15 2016, at 12:38 pm, Jorge Bucaran &lt;notifications@github.com&gt;
wrote:  

>

> In [functions/__fisher_path_update.fish](https://github.com/fisherman/fisher
man/pull/157#discussion_r56114412):

>  
>  
>     > @@ -5,7 +5,7 @@ function __fisher_path_update -a path

>     >          git rebase --abort

>     >          git fetch --quiet origin master

>     >          git reset --quiet --hard FETCH_HEAD

>     > -        git clean --quiet -d --force

>     > +        git clean --quiet -dx --force

>  
>

> I see. With -x we basically make the repo the same as whatever was upstream.
Correct?

>

> —  
You are receiving this because you authored the thread.  
Reply to this email directly or view it on GitHub:  
<https://github.com/fisherman/fisherman/pull/157/files#r56114412>![](https://g
ithub.com/notifications/beacon/ABlxkdj7nHnC2q5N4e54bnh7dyxU-
M07ks5ptje3gaJpZM4HwrV6.gif)